### ISSUE TITLE
Overall formatting stuff

### ISSUE THREAD

**Issue by panunu** [panunu/metropolia-thesis-latex#17]
I will just list miscellaneous issues. Not sure if some of these are already fixed. 

Here we go!

--- 

**Basic usage**

~We should separate the template from the actual content. Now it is quite hard to update the template in the process of writing. Do not really remember is there something which prevents doing this~ :-)

**Translations**

- "Listaus" might be "Koodiesimerkki". 
- ~Instead of "Opinnäytetyö" use "Insinöörityö".~

**Formatting (easy to fix)**

- Captions (e.g. under images) should be in smaller font than the default text.
- Same for formulas.

**Formatting (harder to fix)**

```
\begin{align}
I&=\frac{1}{2}\cdot\sum z_i^2c_i \\
z_i&= \tn{ionin varausluku} \\
c_i&= \tn{ionin konsentraatio}
\end{align}
```
In this `align`, the lines `z_i` and `c_i` should not contain line numbers (at least when there is no reference). Also instead of the `=` there should be a Finnish word "on". This complicates things as `align` uses the `=` character to align stuff. 

Images should _never_ cut sentences (when page changes).

If a thesis only has a single appendix, there should be no number prefix. So instead of **1 This is the title of appendix** it should be just **This is the title of appendix**.

**Comment by laam4** [comment]
But not all theses are "Insinöörityö", mine wasn't.

**Comment by patrick-ausderau** [comment]
Please split them into many issues ;) So we can fix them one by one.
* Agree that we should separate content from template with `\input` and/or `\include`
  * And eventually split the template too (e.g. the "formatting" stuff in one file, the "glossary" in another, "title" page somewhere else,...). If that makes sense?
* Image cutting paragraph is "too hard to fix" (at least with the [basic float flags](https://tex.stackexchange.com/questions/39017/how-to-influence-the-position-of-float-environments-like-figure-and-table-in-lat)). I usually give the instruction to let them float till the text is final and manually move them.

**Comment by patrick-ausderau** [comment]
Fixed the "basic usage" (in commit https://github.com/panunu/metropolia-thesis-latex/commit/1c8e73953cfd03da162022d61c7fb55db62d73c3 and couple of  previews)

**Comment by patrick-ausderau** [comment]
Fixed Koodiesimerkki in 5fc4e70ae2ec65b5414302450ffa1b65e2dd9e51 and caption size in 419c8cd05339aea539c0353604714a9827465e22

**Comment by patrick-ausderau** [comment]
All the pending points have been move to separated issues
